yt_dlp/extractor/appleconnect.py

```diff
@@ -1,47 +1,92 @@
 from .common import InfoExtractor
-from ..utils import ExtractorError, str_to_int
+from ..utils import (
+    ExtractorError,
+    float_or_none,
+    parse_resolution,
+    qualities,
+    url_or_none,
+)
+from ..utils.traversal import traverse_obj
 
 
 class AppleConnectIE(InfoExtractor):
-    _VALID_URL = r'https?://itunes\.apple\.com/\w{0,2}/?post/(?:id)?sa\.(?P<id>[\w-]+)'
+    IE_NAME = 'apple:music:connect'
+    IE_DESC = 'Apple Music Connect'
+
+    _HEADERS = {
+        'Authorization': 'Bearer eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IldlYlBsYXlLaWQifQ.eyJpc3MiOiJBTVBXZWJQbGF5IiwiaWF0IjoxNzQ2NjM3MTY2LCJleHAiOjE3NTM4OTQ3NjYsInJvb3RfaHR0cHNfb3JpZ2luIjpbImFwcGxlLmNvbSJdfQ.ONPUnh6UMOJ1VWujIxxWuTdi2ueBAM01B8xMg4NkNy9mdE_C1Y15-xKGoZ6Qg6mgC-ZMdfFHt5Xf4hL4X4-lMw',
```

Comment:
This expires in 2 months. Can't we extract it?

Reply:
Looks like the JWT has already rotated. It's now `eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IldlYlBsYXlLaWQifQ.eyJpc3MiOiJBTVBXZWJQbGF5IiwiaWF0IjoxNzQ4MDIwNTQzLCJleHAiOjE3NTUyNzgxNDMsInJvb3RfaHR0cHNfb3JpZ2luIjpbImFwcGxlLmNvbSJdfQ.gaSZG9M4qQ5Wy123cDr-BX4qgQ1ZHBuaChm4lMu7TC5v65mNpfXMYhfUuuNIXluhmiUYKIRmC0qzG0I2bq8dXg` which is hardcoded in https://music.apple.com/assets/index-82896190.js

Notable, the first part (`eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IldlYlBsYXlLaWQifQ`, which includes the key id `WebPlayKid`) is the same